sim/sim2sim.py

```diff
@@ -252,6 +252,7 @@ def run_mujoco(
                     "joint_pos": cur_pos_obs.astype(np.float32),
                     "joint_vel": cur_vel_obs.astype(np.float32),
                     "prev_actions": actions.astype(np.float32),
+                    "curr_actions": target_q.astype(np.float32),
```

Comment:
current actions are actions listed below

Reply:
addressed in [df09d0d](https://github.com/kscalelabs/sim/pull/116/commits/df09d0dbca67ef03385babc5a02c9c8aed64555f)

Reply:
This is wrong, you are now saving latest actions and positions (target_q which are scaled actions).

Reply:
okay should be fixed in [0eb70d3](https://github.com/kscalelabs/sim/pull/116/commits/0eb70d3f0eb9f990abdc65cef15ba2b5859d121e)

i renamed `actions` to `prev_actions` since that is what it is being used as, and use `curr_actions` for the output of the model. at the end of the inner loop iteration, I set `prev_actions = curr_actions`.